update to latest revision (r1449) from
https://svn.freepascal.org/svn/fpcdocs/trunk

This fixes a generation error due to the missing file linuxex/ex71.pp.
Index: fpcbuild-3.0.4/fpcdocs/fclres.tex
===================================================================
--- fpcbuild-3.0.4.orig/fpcdocs/fclres.tex
+++ fpcbuild-3.0.4/fpcdocs/fclres.tex
@@ -41,7 +41,7 @@
 \title{Free Pascal Resource support (FCL-res): \\ Reference guide.}
 \label{rtl}
 \docdescription{Reference guide for FCL-res units.}
-\docversion{2.6}
+\docversion{3.0.4}
 \input{date.inc}
 \author{Giulio Bernardi}
 \maketitle
Index: fpcbuild-3.0.4/fpcdocs/internal.tex
===================================================================
--- fpcbuild-3.0.4.orig/fpcdocs/internal.tex
+++ fpcbuild-3.0.4/fpcdocs/internal.tex
@@ -38,7 +38,7 @@
 \begin{document}
 \title{Free Pascal\\ Compiler documentation}
 \docdescription{Compiler documentation for \fpc, version \fpcversion}
-\docversion{2.6}
+\docversion{3.0.4}
 \input{date.inc}
 \author{Micha\"el Van Canneyt\\Florian Kl\"ampfl}
 \maketitle
Index: fpcbuild-3.0.4/fpcdocs/linux.xml
===================================================================
--- fpcbuild-3.0.4.orig/fpcdocs/linux.xml
+++ fpcbuild-3.0.4/fpcdocs/linux.xml
@@ -225,7 +225,6 @@ On error, -1 is returned to the parent,
 <seealso>
 <link id="#rtl.baseunix.FpFork"/>
 </seealso>
-<example file="linuxex/ex71"/>
 </element>
 
 <!-- unresolved type reference Visibility: default -->
Index: fpcbuild-3.0.4/fpcdocs/ref.tex
===================================================================
--- fpcbuild-3.0.4.orig/fpcdocs/ref.tex
+++ fpcbuild-3.0.4/fpcdocs/ref.tex
@@ -47,7 +47,7 @@
 \begin{document}
 \title{Free Pascal\\ Reference guide}
 \docdescription{Reference guide for Free Pascal, version \fpcversion}
-\docversion{3.0.2}
+\docversion{3.0.4}
 \input{date.inc}
 \author{Micha\"el Van Canneyt}
 \maketitle
Index: fpcbuild-3.0.4/fpcdocs/units.tex
===================================================================
--- fpcbuild-3.0.4.orig/fpcdocs/units.tex
+++ fpcbuild-3.0.4/fpcdocs/units.tex
@@ -49,7 +49,7 @@
 %
 \title{Free Pascal supplied units : \\ Reference guide.}
 \docdescription{Reference guide for standard Free Pascal units.}
-\docversion{2.6}
+\docversion{3.0.4}
 \input{date.inc}
 \author{Micha\"el Van Canneyt\\ Florian Kl\"ampfl}
 \maketitle
